docs/src/modules/java/pages/http-endpoints.adoc

```diff
@@ -67,6 +67,32 @@ Additionally, the request body parameter can be of the following types:
  * `akka.http.javadsl.model.HttpRequest` for a low level, streaming representation of the entire request
     including headers. See xref:http-endpoints.adoc#_low_level_requests[] below for more details
 
+=== Request headers ===
```

Comment:
Move it here, I think this is still a basic interaction with endpoints.

Reply:
It's not what you immediately need to build a HTTP API, so I think it was good that it at least is further down the page and not before doing basics like how to return something. Maybe keep in basic block but last?

Reply:
My ordering idea was:
 - basic stuff for request
 - basic stuff for response

Reply:
Ok fair enough